llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -1024,6 +1058,7 @@ void CSProfileGenerator::postProcessProfiles() {
     CSConverter.convertCSProfiles();
     FunctionSamples::ProfileIsCS = false;
   }
+  filterAmbiguousProfile(ProfileMap);
```

Comment:
why do we have it both here and in `postProcessProfiles `

Reply:
The `postProcessProfiles ` is not a shared function for non-cs and cs, i,e, we have `CSProfileGenerator::postProcessProfiles()` and `ProfileGenerator::postProcessProfiles() `.